README.md

```diff
@@ -537,6 +537,47 @@ else
 end
 ```
 
+#### Add-Ons
```

Comment:
Can you add description for every add-on (you can copy them from the api-refs)?

Reply:
Just a human-plain-english description. Not everybody goes to the api-refs or even docs to check it out. Usually people just read readme, and that's it for them. If they don't get it from here, they won't get it anywhere else. :-(